colpali_engine/models/idefics3/colidefics3/processing_colidefics3.py

```diff
@@ -27,23 +26,35 @@ def image_token_id(self) -> int:
     def process_images(
         self,
         images: List[Image.Image],
+        contexts_prompts: Optional[List[str]] = None,
     ) -> BatchEncoding:
         """
         Process images for ColIdefics3.
         """
         texts_doc: List[str] = []
         images = [[image.convert("RGB")] for image in images]
 
-        for _ in images:
-            messages_doc = [
-                {
-                    "role": "user",
-                    "content": [
-                        {"type": "text", "text": "Describe the image."},
-                        {"type": "image"},
-                    ],
-                },
-            ]
+        for i, _ in enumerate(images):
+            if contexts_prompts is not None:
```

Comment:
btw, for idefics, this is not consitent with the behaviour we chose for the qwen models... we should probably remove this chat template thing and writyey the template by hand like we are doinfg for qwen so that everything stays consistent